### ISSUE TITLE
Improve error messages

### ISSUE THREAD

**Issue by ctmay4** [imsweb/staging-client-java#7]
When an ERROR endpoint is reached, we display a very generic message if the the endpoint itself does not specify one:

```
Processing resulted in an error
Processing resulted in an error
Processing resulted in an error
The input mapping 'ss77' does not exist for ss_codes
Processing resulted in an error
The input mapping 'ss2000' does not exist for ss_codes
```

The old C DLL displayed more detailed error messages

```
Lookup of codes (Tispu, N1, M1) in AJCC TNM 6 Stage returns "ERROR".
AJCC stage lookup failed.
Lookup of codes (Tispu, N1, M1) in AJCC TNM 7 Stage returns "ERROR".
AJCC stage lookup failed.
Lookup of codes (IS, RN, D) in Summary Stage returns "ERROR".
SEER Summary Stage 1977 lookup failed.
Lookup of codes (IS, RN, D) in Summary Stage returns "ERROR".
SEER Summary Stage 2000 lookup failed.
```

The library should improve this by adding the table and inputs like the old library provided.

**Comment by ctmay4** [comment]
The two messages look like this now:

```
Match not found in table 'table_id' for inputs (val1,val2,val3)
Matching inputs (val1,val2,val3) in 'table_id' resulted in an error
```

I would prefer to put the table name but it is not currently available since it is not in the interface.  I may consider moving it.

**Comment by ctmay4** [comment]
This was released in v1.3
